test/query/query.test.ts

```diff
@@ -121,6 +122,34 @@ describe('query/query', () => {
       });
     });
 
+    describe('rank', () => {
+      const q: Query = {
```

Comment:
Why isn't this q a part of `it`.  It's not reused so you shouldn't pollute the outer scope.